navit/main.c

```diff
@@ -86,7 +86,20 @@ void setenv(char *var, char *val, int overwrite) {
  * ':'  is replaced with NAVIT_PREFIX
  * '::' is replaced with NAVIT_PREFIX and LIBDIR
  * '~'  is replaced with HOME
+ * ';'  is replaced with USERPROFILE for Windows
 */
+
+#ifdef HAVE_API_WIN32_BASE && !HAVE_API_WIN32_CE
```

Comment:
Should we rather use the existing array that is supposed to index the variable depending on the OS.
If we need two Win32 entries (one WinCE, one Win32), maybe we can expand the array to 6 columns instead of 5.
This would avoid the `#ifdef` preprocessor directive than inherently forces to duplicate code.